lib/quick_travel/cache.rb

```diff
@@ -10,6 +10,8 @@ def self.cache(key, cache_options = {})
       cached_value = cache_store.read(key)
       return cached_value unless cached_value.nil?
       return nil unless block_given?
+      cache_options ||= {}
+      cache_options[:expires_in] = 1.day unless cache_options.key?(:expires_in)
```

Comment:
Point taken, but the above, where expires_in is passed in, would be setting the value to the same value, which doesn't feel right.

What I'm trying to say is how it reads currently, that is "if no expires_in" is set, set it to a day

It's perhaps not clear (and I'm not sure how to improve) that you may want to pass in expires_in: nil to never cache. This is deliberate feature